pymobiledevice3/lockdown.py

```diff
@@ -118,6 +120,100 @@ def __init__(self, udid=None, client_name=DEFAULT_CLIENT_NAME):
         self.paired = True
         return
 
+    def forward(self, src_port: int, dst_port: int):
+        """
+        forward each connection from given local machine port to remote device port
+        """
+        # dictionaries containing the required maps to transfer data between each local
+        # socket to its remote socket and vice versa
+        remote_to_local = {}
+        local_to_remote = {}
+
+        # create local tcp server socket
+        server_socket = socket.socket()
+        server_socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        server_socket.bind(('0.0.0.0', src_port))
+        server_socket.listen(200)
+        server_socket.setblocking(False)
+
+        inputs = [server_socket]
+
+        local_connection = None
+        remote_connection = None
+
+        while inputs:
+            # will only perform the socket select on the inputs. the outputs will handled
+            # as synchronous blocking
+            readable, writable, exceptional = select.select(inputs, [], inputs)
+
+            for s in readable:
+                if s is server_socket:
```

Comment:
Maybe split the following parts to functions?

Reply:
ok